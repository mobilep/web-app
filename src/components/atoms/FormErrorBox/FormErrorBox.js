import React from 'react';
import { ErrorBox } from '..';
import { WarningIcon } from '../../../icons';

const FormErrorBox = (props) => {
	return (
		<ErrorBox {...props} icon={<WarningIcon />} />
	);
};

export default FormErrorBox;
