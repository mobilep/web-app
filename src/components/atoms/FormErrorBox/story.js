import React from 'react';
import { storiesOf } from '@storybook/react';
import { text } from '@storybook/addon-knobs/react';
import FormErrorBox from './FormErrorBox';

storiesOf('1. Atoms / FormErrorBox', module)
	.add('FormErrorBox', () => {
		const message = text('error text', `Uh no, we don't recognise that email address`);
		return (
			<FormErrorBox {...{ message }} />
		);
	});
