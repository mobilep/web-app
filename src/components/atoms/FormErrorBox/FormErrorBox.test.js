import React from 'react';
import { shallow } from 'enzyme';

import FormErrorBox from './FormErrorBox';

describe('Atom - FormErrorBox', () => {

	it('Should render', () => {
		const wrapper = shallow(<FormErrorBox message='Message' />);
		expect(wrapper).toMatchSnapshot();
	});

});

