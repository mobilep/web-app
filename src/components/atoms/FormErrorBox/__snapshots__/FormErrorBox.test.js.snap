// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atom - FormErrorBox Should render 1`] = `
<ErrorBox
  icon={<WarningIcon />}
  message="Message"
/>
`;
